content/en/docs/best-practices/common.md

```diff
@@ -7,7 +7,7 @@ description: Common recommendations and suggestions to distribute solutions with
 
 ## Overview
 
-Any recommendation or good practice suggested by the Kubernetes community such as to develop [Operator pattern][operator-pattern] solutions or to manage them are good recommendations for who is looking for to build the operator projects and distribute them with OLM. Also, see [Operator Best Practices][operator-best-practices].
+Any recommendation or good practice suggested by the Kubernetes community such as to develop [Operator pattern][operator-pattern] solutions or to manage them are good recommendations for who is looking for to build the operator projects and distribute them with OLM. Also, see [Operator Best Practices][operator-best-practices] and ensure that you check out [Running on Cluster][running-on-cluster].
```

Comment:
Real minor nit, but if we're presumably referencing section titles in the OLM/SDK documentation, should we update `Running on Cluster` -> `Running On-Cluster`?

/lgtm

Reply:
Done.